### ISSUE TITLE
What is the meaning of entity names?

### ISSUE THREAD

**Issue by Ask-sola** [urchade/GLiNER#230]
Hello, I read your paper and I noticed that the final loss iteration used a labeling method to calculate binary cross-entropy loss. If I understand correctly, this seems to be related to the relationship between spans and labels. So, what role does the entity name encoding play in the loss training process?
